exercise-02/exercise-02.js

```diff
@@ -16,7 +16,18 @@
  * ```
  */
 const isPalindrome = (value) => {
-    throw new Error()
+  try{
+  let reg = /[\W_]/g;
+  let lowerCase = value.toLowerCase().replace(reg, '')
+  let vArray = lowerCase.toLowerCase().split("").reverse().join("");
+ 
+  const check = (lowerCase === vArray) ? true : false;
+  return check;
```

Comment:
No need for a variable just to return it on the next line
`return (lowerCase === vArray) ? true : false;` would be enough